packages/fxa-shared/l10n/branding.ftl

```diff
@@ -20,6 +20,7 @@
 # "accounts" can and should be localized, "Firefox" must be treated as a brand.
 # 'Firefox accounts' refers to the service
 -product-firefox-accounts = Firefox accounts
+-product-mozilla-accounts = Mozilla accounts
```

Comment:
This really needs a comment otherwise there's not going to be consistency in localization
```suggestion
# "account" can and should be localized, "Mozilla" must be treated as a brand. Singular "Mozilla account" should be used in most cases.
-product-mozilla-account = Mozilla account
# "accounts" can and should be localized, "Mozilla" must be treated as a brand. Plural "Mozilla accounts" is used when referring to something affecting all Mozilla accounts, not just the individual's account.
-product-mozilla-accounts = Mozilla accounts
```